Content.IntegrationTests/Tests/CargoTest.cs

```diff
@@ -149,6 +154,79 @@ public async Task NoStaticPriceAndStackPrice()
         await pair.CleanReturnAsync();
     }
 
+    /// <summary>
+    /// Tests to see if any items that are valid for cargo bounties can be sliced into items that
+    /// are also valid for the same bounty entry.
+    /// </summary>
+    [Test]
+    public async Task NoSliceableBountyArbitrageTest()
+    {
+        await using var pair = await PoolManager.GetServerClient();
+        var server = pair.Server;
+
+        var testMap = await pair.CreateTestMap();
+
+        var entManager = server.ResolveDependency<IEntityManager>();
+        var mapManager = server.ResolveDependency<IMapManager>();
+        var protoManager = server.ResolveDependency<IPrototypeManager>();
+        var whitelist = entManager.System<EntityWhitelistSystem>();
+        var cargo = entManager.System<CargoSystem>();
+        var sliceableSys = entManager.System<SliceableFoodSystem>();
+
+        var bounties = protoManager.EnumeratePrototypes<CargoBountyPrototype>().ToList();
+
+        await server.WaitAssertion(() =>
+        {
+            var mapId = testMap.MapId;
+            var grid = mapManager.CreateGridEntity(mapId);
+            var coord = new EntityCoordinates(grid.Owner, 0, 0);
+
+            var sliceableEntityProtos = protoManager.EnumeratePrototypes<EntityPrototype>()
+                .Where(p => !p.Abstract)
+                .Where(p => !pair.IsTestPrototype(p))
+                .Where(p => p.Components.ContainsKey("SliceableFood"))
```

Comment:
use TryGetComponent since content components are not secure to get like this because the name can easily change